.cursor/plans/homepage_cache_tag_propagation_c503aef3.plan.md

```
@@ -0,0 +1,64 @@
+    dependencies:
+      - add-contentfeed-options
+  - id: tag-contentfeed-route
+    content: "Update /api/contentfeed route GET handler to pass tags: ['page:home']."
+    status: completed
+    dependencies:
+      - add-contentfeed-options
+---
+
+# Add `page:home` cache tag to homepage feed fetching
+
+## What we’re changing
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)